src/styles/base/_colors.min.css

```diff
@@ -0,0 +1 @@
+:export{primary1Color:#0F94a7}:export{primary2Color:#485566}:export{accent1Color:#91C383}:export{alternateTextColor:#fff}:export{borderColor:#DADDE0}:export{textColor:#485566}
```

Comment:
Fixed - also fixed the problem where the editor was generating compiled sass. =) Shouldn't be an issue anymore.